hal_api-rails.gemspec

```diff
@@ -30,4 +30,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.10"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "minitest"
+  spec.add_development_dependency "kaminari"
+  spec.add_development_dependency "pundit"
```

Comment:
Is the pundit/current_user/authorization stuff I see in here necessary?  (Asking because some of it is just commented out).